docs/adrs/003-bugzilla-response-codes.md

```diff
@@ -0,0 +1,100 @@
+# Bugzilla disables webhooks after too many errors
+
+- Status: proposed
+- Date: 2023-27-11
+
+Tracking issue: 
+
+## Context and Problem Statement
+When bugzilla receives too many error responses from JBI, it stops triggering webhook calls for the entire project causing data to stop syncing. Frequently these errors are due to JBI being unable to process a payload due to errors in configuration (or incomplete configuration) in Jira or a mismatch of data for a single bug. 
```

Comment:
> If we configure a webhook per product, it will only stop sending events for this product and won't affect the rest (See https://github.com/mozilla/jira-bugzilla-integration/issues/21).

This is the way it is right now in production -- we have many webhooks set up for each Bugzilla product (scoped to `ANY` component).

<img width="337" alt="image" src="https://github.com/mozilla/jira-bugzilla-integration/assets/20747904/5510d9c0-d9ab-431f-b912-3b1546af9f6b">